.github/workflows/spec-author.yml

```
@@ -0,0 +1,152 @@
+
+jobs:
+  author:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: 出向白名单（仅 github + provider，INV-06）
+        uses: step-security/harden-runner@05e31511f85b41b11d1cf0ef85d0992719546e2c # v2.21.0
+        with:
+          egress-policy: block
+          allowed-urls:
+            - github.com
```

actionlint (1.7.12)

[error] expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)